src/components/POLine/POLineDetails/FieldOrderFormat.js

```diff
@@ -41,17 +42,25 @@ class FieldOrderFormat extends Component {
     dispatch(change('cost.quantityElectronic', ''));
     dispatch(change('cost.listUnitPriceElectronic', ''));
     dispatch(change('cost.listUnitPrice', ''));
+    dispatch(change('locations', [{}]));
```

Comment:
I'm not sure if it's needed. Previous option I'd like better, it was something like `initialValues.locations = [{}]`.

Reply:
but previous implementation doesn't allow cleaning locations field after changing order format